git_machete/utils.py

```diff
@@ -257,3 +257,8 @@ def slurp_file_or_empty(path: str) -> str:
             return file.read()
     except IOError:
         return ''
+
+
+def perform_interactive_slide_out(cmd: str) -> bool:
+    high_level_commands = {'traverse', 'status'}
```

Comment:
Yeah, let's stick to these two for now for simplicity... time will tell if any changes need to be done here

Reply:
Maybe actually I'd call the var `interactive_slide_out_safe_commands`, coz there are other commands that can be considered _high-level_